test/multisigwallet/confirm_transaction.js

```diff
@@ -0,0 +1,210 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Test: MultiSigWallet::confirmTransaction
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const BN = require('bn.js');
+const utils = require('../test_lib/utils.js');
+const { Event } = require('../test_lib/event_decoder.js');
+
+const MultiSigWallet = artifacts.require('MultiSigWallet');
+
+contract('MultiSigWallet::confirmTransaction', async () => {
+    contract('Negative testing for input parameters.', async (accounts) => {
```

Comment:
Also,add test case for empty transaction id.

Reply:
I believe unit test cases should cover all the possible scenarios.So,checking wrong type should be covered.

Reply:
@abhayks1 transactionID == 0 is a valid case. There could be transaction with that ID. I am not following what is your proposal?

@gulshanvasnani Should we check that it should not be false, an empty array, an empty struct, etc? I mean, checking the wrong types will bloat test cases and it's not what we should test.

Reply:
We do check for 'empty' or 'null' values in case of bytes or bytes32 datatype.
So,i was thinking of having the same for uint256 datatype.
But,if you think it will bloat test cases,not a problem.